addons/website/static/src/js/widgets/theme.js

```diff
@@ -325,18 +325,22 @@ var ThemeCustomizeDialog = Dialog.extend({
                         var xmlid = $item.data('xmlid');
 
                         var renderingOptions = _.extend({
-                            string: $item.attr('string') || data.names[xmlid.split(',')[0].trim()],
+                            string: $item.attr('string') || xmlid && data.names[xmlid.split(',')[0].trim()],
                             icon: $item.data('icon'),
                             font: $item.data('font'),
                         }, $item.data());
 
+                        var checked = ($item.data('widget') === 'auto'
```

Comment:
Use the `widgetName` variable instead of reading data again. Also use ternary form which would be more efficient and correct:
```javascript
var checked = widgetName === 'auto' ?
    self.style.blablabla === blablabla : 
    xmlid && blablabla;
```
But in fact, your condition is still not ok, see before: when no xmlid, it was checked. So it is rather:
```javascript
var checked = widgetName === 'auto' ?
    self.style.blablabla === blablabla : 
    !xmlid || blablabla;
```
right ?